src/components/login/LogoutButton.tsx

```diff
@@ -1,27 +1,21 @@
 import { useTranslation } from 'next-i18next'
 import { FunctionComponent, useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
 import { logout } from '../../context/actions'
 import { useUserDispatch } from '../../context/user-info'
 import Button from '../Button'
-import FeedbackMessage from '../FeedbackMessage'
 
 const LogoutButton: FunctionComponent = () => {
   const { t } = useTranslation()
   // Using state to prevent user repeatedly initating fetches
   const [clicked, setClicked] = useState(false)
-  const [error, setError] = useState('')
   const dispatch = useUserDispatch()
 
   // Only make a request on first click
   useEffect(() => {
-    if (clicked) { logout(dispatch, setError) }
+    if (clicked) { logout(dispatch, toast.error) }
```

Comment:
I think this also has untranslated errors?

Reply:
Maybe, it makes more sense to use `toast.error directly in `logout` or `login` and don't pass it down. I don't think we gain anything from that.

Reply:
I was assuming it has to be used in the component, but actually don't love this pattern I've landed on of passing functions out to the asyncs anyway. Think it would be cleaner to use .then and .catch in the component. Will adjust when I fix these to use translation strings tomorrow.